jmix-flowui/flowui-themes/src/main/resources/META-INF/resources/themes/jmix-lumo/component/jmix-twin-column.css

```diff
@@ -0,0 +1,116 @@
+jmix-twin-column::part(required-indicator) {
```

Comment:
Are you sure that this style is still needed after you applied input-field styles?